patch/attiny85.yaml

```diff
@@ -98,3 +98,40 @@ CPU:
     _modify:
       PLOCK:
         access: read-only
+EXINT:
+  # This doesn't work:
+  # _modify:
+  #   PCMSK:
+  #     _write_constraint: [0, 63]
```

Comment:
Yeah, the version of `svdtools` which we are using at the moment doesn't seem to support updating the write constraint of whole registers :/

Let's keep it the way you wrote it so we can uncomment the block once we upgrade to a version of `svdtools` that does support it.